.eslintrc

```diff
@@ -3,150 +3,140 @@
   "env": {
     "browser": true,
     "node": true,
-    "jasmine": true
+    "es6": true
   },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "extends": [
+    "prettier",
+    "prettier/flowtype",
+    "prettier/react"
```

Comment:
Haven't thoroughly investigated the rule changes you've done, but would it make sense to just move to airbnb's preset (with some additions/overrides)?